docs/sources/shared/back-up/back-up-grafana.md

```diff
@@ -17,7 +17,7 @@ Copy Grafana configuration files that you might have modified in your Grafana de
 
 The Grafana configuration files are located in the following directories:
 
-- Default configuration: `$WORKING_DIR/conf/defaults.ini`
+- Default configuration: `$WORKING_DIR/conf/config.ini`
```

Comment:
This change is incorrect as the Default configuration file is `defaults.ini`. 

- Suggest including the `Do not change this file.`

- In this section, I also suggest including the sentence added on the `Upgrade section`:

 `For more information on where to find configuration files, refer to [Configuration file location](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#configuration-file-location).`
 
- When this section refers to the `Custom configuration`, it should indicate `custom.ini` or `grafana.ini`

Reply:
Sorry I got confused about your Slack message about updating this section. I'll make an update tomorrow, but please just correct the update to what you think it should be if it's not right.